dockstore-webservice/src/main/resources/migrations.1.9.0.xml

```diff
@@ -28,4 +28,12 @@
             <where>type='ADD_VERSION_TO_ENTRY' and versionid is null</where>
         </delete>
     </changeSet>
+    <changeSet author="aduncan" id="1580931394009">
```

Comment:
can use meaningful ids to make debugging in the DB easier

Reply:
Do you mean setting default value to "n/a"?

Reply:
Nevermind I see your other comment.